scripts/joinmarket-qt.py

```diff
@@ -2305,6 +2305,15 @@ def onTabChange(i):
 tabWidget.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
 mainWindow.setCentralWidget(tabWidget)
 tabWidget.currentChanged.connect(onTabChange)
+
+
+# TODO: Remove these testing code
+from jmui import Ui_openWalletDialog
+openWalletDialog = QDialog()
+ui = Ui_openWalletDialog()
+ui.setupUi(openWalletDialog)
+openWalletDialog.show()
```

Comment:
All the actual logic, like what happens when the user click a button, etc., will all be written at a separate place like in this `joinmarket-qt.py` file, instead of in the python UI file (like `open_wallet_dialog.py`).